website/docs/developer/development-guide/code/postgres-tests.md

```diff
@@ -0,0 +1,30 @@
+---
+id: postgres-tests
+title: Running Postgres Tests
+description: Instructions on running 'postgres' tests.
+---
+
+## Preparing to Run Postgres Tests
+Test files ending in '_postgres.py' require a postgres database to be accessible.
+
+1. Make sure you have Docker running on your machine.
+1. Open a command prompt and run
+    ```commandline
+    docker run -it --rm --name some-postgres -v [path to repository]/database/ddl/base:/docker-entrypoint-initdb.d/ -p 5432:5432 -e POSTGRES_PASSWORD=postgres postgres
```

Comment:
should change out the `POSTGRES_PASSWORD=postgres` with an environment variable like so: `POSTGRES_PASSWORD=$PG_PASSWORD` or something similar.

Reply:
This step occurs before these environment variables exist.

Reply:
Done